templates/repo/settings/options.tmpl

```diff
@@ -753,6 +746,42 @@
 				</div>
 			{{end}}
 
+			{{if not .Repository.IsFork}}
+				<div class="divider"></div>
+
+				<div class="item">
+					<div class="ui right">
+						<button class="ui basic red show-modal button" data-modal="#repo-visibility-modal">
+							{{.locale.Tr "repo.visibility_helper" | Safe}}
```

Comment:
It's not good idea to combine label using different translation parts as in some languages it could be that private|public could be in the middle of the text. It would be better to make them as two different translatable keys with full label text

Reply:
done